### DISCUSSION TITLE
Support for other languages

### DISCUSSION THREAD

**Discussion by DIABL06666** [jwebmeister/tacspeak#33] [Q&A] [answered]
Is it possible to add the Polish language?

**Comment by jwebmeister** [comment, marked as answer]
If there's a Polish (or other) language Kaldi Active Grammar model, Tacspeak should already support it AFAIK. 

I'm not personally planning to make any new models in any language other than English.

I'm _unsure_ if it's possible (without significant effort + reverse engineering) to train a new model in another language, because training a new model to work with Kaldi Active Grammar requires a base model, which I believe there's only one of, which is in English. Keyword is _unsure_, it might not be an issue.
You can look in the Kaldi Active Grammar GitHub repository (can also look at the kaldi_ag_training repo) if you're interested in more info, link's in the README.md.
